remappings.txt

```diff
@@ -0,0 +1,2 @@
+forge-std/=lib/forge-std/src/
+@prb/test/=lib/prb-test/src/
```

Comment:
[this is a more advanced version of the `forg-std/Test` contract](https://github.com/PaulRBerg/prb-test)